cmake/GenerateBuiltins.cmake

```diff
@@ -503,3 +503,133 @@ function (generate_builtins)
         ${DISPATCH_BUILTIN_CPP_FILES} ${COMMON_BUILTIN_CPP_FILES} ${TARGET_BUILTIN_CPP_FILES})
     add_dependencies(builtin builtins-cpp)
 endfunction()
+
+# Define custom commands to generate bitcode for a specific generic target and
+# a corresponding cpp file. All they are generated from builtins/generic.ispc
```

Comment:
I suggest clarifying that it's not just cpp, but a "cpp wrapper for bitcode". The reason I'm picking about, generic targets used to mean "target that produces C++ code", so disambiguating this important for anyone familiar with old thing.

Reply:
Done